src/test/java/net/datafaker/providers/base/AviationTest.java

```diff
@@ -18,12 +18,12 @@ void flight_ICAO() {
 
     @Test
     void flight_IATA() {
-        assertThat(aviation.flight("IATA")).matches("[A-Z]{2}[0-9]+");
+        assertThat(aviation.flight("IATA")).matches("[A-Z|0-9]{2}[0-9]+");
```

Comment:
Should the character class really include pipe?

Reply:
Let me rephrase, the square brackets shouldn’t include pipe. In either test.

Reply:
Sorry, I am confused by you comment. I altered the regexp because a flight ID could be like 2B2345 or A97887. Previous data set had no such airlines codes (2B, A9 etc) so the regexp was ok. Now, since a IATA airline code could include a digit the original test may fail.

Reply:
Ah, sorry, I got your point. Fixed.